### PR TITLE
Update RIF to 1.7.0

### PR DESCRIPTION
**Pull request by bsavery**
### PURPOSE
Update RIF to 1.7.0

### CODE REVIEW

**Comment by bsavery** [comment]
@Dizzyspb QA tested 1.7.0?

**Review by bnagirniak — CHANGES_REQUESTED** [review]
Build is broken here.
Some months ago we were planning to update RIF 1.6.2 and I had same broken build which I fixed there and did some refactoring in build process to simplify it. I'd like to integrate that changes here and fix build.

**Comment by bnagirniak** [comment]
I cherry picked commits which I did earlier during RIF 1.6.2 integration. So here:
1. Fixed RIF build by adjusting cmd_tools/create_sdk.py and src/bindings/pyrpr/rpr.py.
2. Refactored and improved loading libs process by adding RPR_BLENDER_DEBUG env variable which defines where to load libs. This fixed also log error during plugin running in debug mode.

File: cmd_tools/create_sdk.py
Comment (bsavery) [inline cmd_tools/create_sdk.py:112]: This seems like it could be made more generic no? So we don't need to update this version with each release?

File: src/bindings/pyrpr/src/pyrpr.py
Comment (bsavery) [inline src/bindings/pyrpr/src/pyrpr.py:105]: Why is this change needed in a RIF change?

File: src/bindings/pyrpr/src/pyrpr.py
Comment (bsavery) [inline src/bindings/pyrpr/src/pyrpr.py:200]: Not used anymore?

**Review by bsavery — COMMENTED** [review]
Lots of questions.

File: cmd_tools/create_sdk.py
Comment (bnagirniak) [reply cmd_tools/create_sdk.py:112]: yes, will do

File: src/bindings/pyrpr/src/pyrpr.py
Comment (bnagirniak) [reply src/bindings/pyrpr/src/pyrpr.py:200]: It was never used. Long time ago I wrote it and planed to use. This is just cleanup.

File: src/bindings/pyrpr/src/pyrpr.py
Comment (bnagirniak) [reply src/bindings/pyrpr/src/pyrpr.py:105]: I did that change as a code cleanup several months ago and just cherry picked commits.

**Review by bnagirniak — APPROVED** [review]
Build is ok and seems works also ok on CIS. Waiting for approval from QA.

**Comment by ardraeiss** [comment]
It finally works for me as well, waiting for QA approval.

**Review by Dizzyspb — APPROVED** [review]
RIF 1.7.0 integration is tested by QA. Approved for merge.

**Pull request merged**
